test/extensions/filters/http/csrf/csrf_filter_test.cc

```diff
@@ -0,0 +1,337 @@
+#include "common/http/header_map_impl.h"
+
+#include "extensions/filters/http/csrf/csrf_filter.h"
+
+#include "test/mocks/buffer/mocks.h"
+#include "test/mocks/http/mocks.h"
+#include "test/mocks/stats/mocks.h"
+#include "test/test_common/printers.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
+using testing::DoAll;
+using testing::InSequence;
+using testing::Invoke;
+using testing::NiceMock;
+using testing::Return;
+using testing::ReturnRef;
+using testing::SaveArg;
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Csrf {
+
+class CsrfFilterTest : public testing::Test {
+public:
+  CsrfFilterConfigSharedPtr setupConfig() {
+    envoy::config::filter::http::csrf::v2::CsrfPolicy policy;
+    policy.mutable_filter_enabled()->mutable_default_value()->set_numerator(100);
```

Comment:
factor out  policy.mutable_filter_enabled()->mutable_default_value() into a temp pointer for brevity?

Reply:
👍 Would it be nicer to factor out only `policy.mutable_filter_enabled()` so I can use it for all three lines?